### ISSUE TITLE
Right justified fixed length fields incorrectly parsed

### ISSUE THREAD

**Issue by GoogleCodeExporter** [unlockha/beanio#1]
```
Right justified fixed length fields are incorrectly parsed.  For example, with 
padding="0" and justify="right", field text "01" may be incorrectly parsed as 
"".


```

Original issue reported on code.google.com by `kevin.s...@gmail.com` on 27 Jan 2011 at 9:58

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 27 Jan 2011 at 10:03
- Added labels: **Usability**

**Comment by GoogleCodeExporter** [comment]
Original comment by `kevin.s...@gmail.com` on 27 Jan 2011 at 10:31
- Added labels: **Milestone-0.9.2**

**Comment by GoogleCodeExporter** [comment]
```
Fixed in release 0.9.2.
```

Original comment by `kevin.s...@gmail.com` on 30 Jan 2011 at 12:08
- Changed state: **Fixed**
- Removed labels: **Usability**
